llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -94,6 +94,7 @@
 #include <bitset>
 #include <cassert>
 #include <cctype>
+#include <cmath>
```

Comment:
What is this needed for?

Reply:
Seems to be for std::log2, but as you noted LLVM's integer log functions should be used instead.